components/ai_chat/resources/page/components/site_title/style.module.scss

```diff
@@ -36,9 +37,50 @@
   align-items: center;
   justify-content: center;
 
+  &:focus-visible {
+    outline: none;
+    box-shadow: var(--leo-effect-focus-state);
+  }
+
   img {
     width: 16px;
     height: 16px;
     object-fit: contain;
   }
 }
+
+.actions {
+  --leo-icon-color: var(--leo-color-text-secondary);
```

Comment:
I believe this prevents the button from having a hover color. Is this intentional?

Reply:
No, it's not intentional. It doesn't inherit from simple `color` so I had to define it. Otherwise here's the default

<img width="462" alt="image" src="https://github.com/brave/brave-core/assets/8665427/65d80762-73e6-4ecd-a88c-da9f7ef0983c">

Reply:
Shouldn't we be using PlainFaint kind of button?

Reply:
didn't know plain-faint existed